frontend/src/k8sTypes.ts

```diff
@@ -47,7 +47,6 @@ export type NotebookAnnotations = Partial<{
   'opendatahub.io/link': string; // redirect notebook url
   'opendatahub.io/username': string; // the untranslated username behind the notebook
   'opendatahub.io/service-mesh': string; // Openshift Service Mesh : determines if mesh configuration should be applied
```

Comment:
Shouldn't we add `opendatahub.io/service-mesh-gw-host` here as well? Or is it only set of the annotations which dashboard is creating?

Reply:
I don't think so - the list here and in [backend/src/types.ts](https://github.com/maistra/odh-dashboard/pull/4/files/45e8586749e4d5b99688177be7381126b9446ba1#diff-86627e5e05bc53054526ea4caa08d7afd5cfdb533adb8b5f2139814454eaa652) are made up only of NotebookAnnotations and `opendatahub.io/service-mesh-gw-host` is a namespace annotation.

Reply:
Maybe worth mentioning now in the parent PR description where is this set